pywinauto/hooks.py

```diff
@@ -0,0 +1,231 @@
+from ctypes import wintypes
+from ctypes import windll, CFUNCTYPE, POINTER, c_int, c_void_p, byref
+import atexit
+
+
+def create_pointer(handler):
+    cmp_func = CFUNCTYPE(c_int, c_int, c_int, POINTER(c_void_p))
+    return cmp_func(handler)
+
+
+class KeyboardEvent:
+    def __init__(self, current_key=None, event_type=None, pressed_key=[]):
```

Comment:
Didn't have too much time to review the whole code. Just my two cents. Don't use list as a default argument. Use
```python
def append_to(element, to=None):
    if to is None:
        to = []
    to.append(element)
    return to
```
More details here: http://docs.python-guide.org/en/latest/writing/gotchas/